enterprise/ha/src/test/java/org/neo4j/ha/TestBranchedData.java

```diff
@@ -200,6 +206,13 @@ public void shouldCopyStoreFromMasterIfBranchedInLiveScenario() throws Throwable
         assertTrue( hasNode( odin, "0-0" ) );
     }
 
+    private BranchMonitor installBranchedDataMonitor( HighlyAvailableGraphDatabase odin )
+    {
+        BranchMonitor monitor = new BranchMonitor();
+        odin.getDependencyResolver().resolveDependency( Monitors.class ).addMonitorListener( monitor );
+        return monitor;
+    }
+
     private void retryOnTransactionFailure( ThrowingAction<Exception> transaction ) throws Exception
```

Comment:
We could use the new `RetryHandler` here. Outside PR scope though, so feel free to ignore.